carshare/settings.py

```diff
@@ -218,8 +219,8 @@ def env_get_list(variable, default):
 SMS_API_KEY = os.environ.get("SMS_API_KEY", None)
 
 # Stripe
-STRIPE_API_KEY = os.environ.get("STRIPE_API_KEY", None)
-STRIPE_PUBLIC_KEY = os.environ.get("STRIPE_PUBLIC_KEY", None)
+STRIPE_API_KEY = "sk_test_51QNWpQ3xFlyrd8hJqbknde1JD8HBAx4gmS6aOO1VPIfnxaIGSKASVrkFuB8wSp0hN1I9u0twZaVOUQjumgocanyM00NUxVgk8s"
+STRIPE_PUBLIC_KEY = "pk_test_51QNWpQ3xFlyrd8hJLfSwaERLEBe0XRPeBfTV5InYkTsrW589oSZDBWHXdHIN6Nw9JdyoLKwdkY1syu5tkISI9j7e00l3boDcB0"
```

Comment:
Oops. Remember to never set these in the file but to always export the environment variable in your local system!